recipes/default.rb

```diff
@@ -12,7 +12,7 @@
 end
 
 # Command to configure w32time service
-manualpeerlist = "#{node['w32time']['ntpserver']},#{node['w32time']['flag']}"
+manualpeerlist = "#{node['w32time']['servers'].join(sep=",#{node['w32time']['flag']} ")},#{node['w32time']['flag']}"
```

Comment:
I think it's complicated, why not doing this instead ? 

`manualpeerlist = node['w32time']['servers'].join(",#{node['w32time']['flag'] ")`